tfe/resource_tfe_workspace.go

```diff
@@ -163,6 +163,7 @@ func resourceTFEWorkspaceCreate(d *schema.ResourceData, meta interface{}) error
 
 		options.VCSRepo = &tfe.VCSRepoOptions{
 			Identifier:        tfe.String(vcsRepo["identifier"].(string)),
+			Branch:            tfe.String(vcsRepo["branch"].(string)),
```

Comment:
does this override the conditional setting of branch on [lines 171-175](https://github.com/terraform-providers/terraform-provider-tfe/pull/220/files#diff-cc7250d176415243a532f4d62525f045R171-R175)? 
it seemed like previously, this was attempting to only set the branch if it was actually set in the config. 
i'm pretty sure it will work either way but i think we might want to pick one way or the other and remove the redundant code.

Reply:
You are right on this one, I missed it on the initial review because the diff was too short :) Good catch!